deepmd/pt/loss/property.py

```diff
@@ -69,34 +78,61 @@ def forward(self, input_dict, model, label, natoms, learning_rate=0.0, mae=False
             Other losses for display.
         """
         model_pred = model(**input_dict)
-        assert label["property"].shape[-1] == self.task_dim
-        assert model_pred["property"].shape[-1] == self.task_dim
+        nbz = model_pred["property"].shape[0]
+        assert model_pred["property"].shape == (nbz, self.task_dim)
+        label["property"] = label[self.property_name]
+        assert label["property"].shape == (nbz, self.task_dim)
+
+        if self.out_std is None:
+            out_std = model.atomic_model.out_std[0][0]
+        else:
+            out_std = torch.tensor(
+                self.out_std, dtype=env.GLOBAL_PT_FLOAT_PRECISION, device=env.DEVICE
+            )
+        if out_std.shape != (self.task_dim,):
+            raise ValueError(
+                f"Expected out_std to have shape ({self.task_dim},), but got {out_std.shape}"
+            )
+
+        if self.out_bias is None:
+            out_bias = model.atomic_model.out_bias[0][0]
```

Comment:
please explain the dimension of `model.atomic_model.out_bias`.  
It it be ntypes x prop_dim? If so, it does not make sense to use a uniform bias for all the `prop_dim` components.

Reply:
The dimension of `model.atomic_model.out_bias` is `nkeys * ntypes * prop_dim`. 
The meaning of first dimension: 
- If fitting type is `energy`, nkeys = 3 (energy,. force, virial)
- If fitting type if `property`, nkeys = 1 (property)
So the shape of `model.atomic_model.out_bias[0][0]`  is (prop_dim,). We use different bias for each `prop_dim` components.